lib/matplotlib/tests/test_legend.py

```diff
@@ -477,6 +477,14 @@ def test_handler_numpoints():
     ax.legend(numpoints=0.5)
 
 
+@image_comparison(baseline_images=['empty_bar_legend'], extensions=['svg'])
```

Comment:
OK, this is failing because your the baseline image is a different size.  But...

1. try not to use svg for this sort of thing because we just rasterize to png anyway.
2. we try not to do image comparisons if possible because they are expensive in terms of time and bandwidth.  Its my undertstanding this crashed before, so all you need for this to pass is for this not to crash (i.e. a smoketest).  Suggest you simply remove the image comaprison

Reply:
I would say just get rid of the image comparison and the test image, I think it's good enough to just check the code runs without any errors.

Reply:
Thank you for your comments. I will keep that in mind. In the future, how would I do image comparisons? I created the image using `save_fig` and it seems to output in a different dimension.

Reply:
I usually run the tests locally (make sure that in `setup.cfg` you have compiled so that freetype is local) and then copy the resulting image into the baseline images...